code/week2/operations.cpp

```diff
@@ -0,0 +1,25 @@
+int add(int a, int b) {
+    return a + b;
+}
+
+int subtract(int a, int b) {
+    return add(a, -b);
+}
+
+int multiply(int a, int b) {
+    int accum = 0;
+    for(int i = 0; i < a; i++) {
```

Comment:
@jgkamat We discussed this in Monday's meeting and decided that it would be fine